cdap-app-fabric/pom.xml

```diff
@@ -265,6 +265,11 @@
           <artifactId>cdap-event-reader-spi</artifactId>
           <version>${project.version}</version>
       </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
```

Comment:
This is assertions lib usually used in tests only. Do we need it for production code?

Reply:
Junit 4.11 brings in hamcrest 1.3. If you need 2.2, please use dependency management in the main pom. This would ensure we don't have two hamcrest versions in the same classpath as this creates strange bugs.

Reply:
added in  dependency management in the main pom